### PR TITLE
feat(core): Add support for floating licenses

### PR DESCRIPTION
**Pull request by csuermann**
This PR updates the license-sdk to v2.5.0 which introduces optional support for floating licenses.

### CODE REVIEW

File: packages/cli/src/commands/start.ts [hunk 1]
```diff
@@ -133,8 +134,13 @@ export class Start extends BaseCommand {
...
 			}
 
-			//finally shut down Event Bus
+			// Shut down Event Bus
 			await eventBus.close();
+
+			// Finally shut down License manager to unclaim any floating entitlements.
+			// Note: In case of above shutdown activities taking too long, the next line might not get executed.
+			// This is fine, as all entitlements will still be claimed on the next startup, which is the desired behavior in most cases.
+			await Container.get(License).shutdown();
```
Comment (netroy) [inline packages/cli/src/commands/start.ts:143]: Can we please still move this to line 100, to give this a higher chance of success?

Comment (csuermann) [reply packages/cli/src/commands/start.ts:143]: I believe it's prudent to perform this action right at the end of the `stopProcess()` function. This approach ensures that enterprise features, which might still be necessary for completing ongoing workflow executions and other tasks, aren't prematurely disabled.

Comment (netroy) [reply packages/cli/src/commands/start.ts:143]: could we notify the license server at the start of `stopProcess`, and not update the in-memory feature flags at all?

Comment (csuermann) [reply packages/cli/src/commands/start.ts:143]: Good idea! That could work. Let me look into that.

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.6.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.6.0)

**Pull request merged**